test/plugin_functional/plugins/core_provider_plugin/kibana.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "core_provider_plugin",
+  "version": "0.0.1",
+  "kibanaVersion": "kibana",
+  "optionalPlugins": ["core_plugin_a", "core_plugin_b", "licensing"],
```

Comment:
I don't see we reference any x-pack plugins here. They are used only if present in the codebase. In the future, we might automate the generation of the optional plugins list.